docs/src/tutorials/bridging_constraint.md

```diff
@@ -177,6 +177,16 @@ function Bridges.added_constrained_variable_types(::Type{SomeBridge{T}}) where {
 end
 ```
 
+!!! warning
+    If you declare the creation of constrained variables, the corresponding 
+    constraint type `SingleVariable` should not be indicated in 
+    `added_constrained_variable_types`. This would restrict the use of the 
```

Comment:
You mean `added_constraint_types` instead of `added_constrained_variable_types` ?

Reply:
Indeed, thanks!